checkstyle-custom/src/test/java/io/vivy/checkstyle/custom/NoLineBreakAfterAssign.java

```diff
@@ -0,0 +1,51 @@
+package io.vivy.checkstyle.custom;
+
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
static imports should go after normal ones

Reply:
fixed